HealthVault.podspec

```diff
@@ -26,4 +26,6 @@ The healthvault-ios-sdk framework simplifies developing apps that use the Micros
   s.xcconfig         = { 'HEADER_SEARCH_PATHS' => '$(inherited) $(SDKROOT)/usr/include/libxml2', 'OTHER_LDFLAGS' => '-lxml2' }
   s.frameworks       = 'UIKit', 'Security', 'MobileCoreServices', 'SystemConfiguration'
 
+  s.dependency 'EncryptedCoreData', '~> 3.1'
```

Comment:
We should probably add this as a subspec, so consumers opt-out of the dependency. We have to think of a decent name for the subspec though... Maybe ThingCache or EncryptedThingCache? I think we should also create a default subspec that does not include the EncryptedCoreData framework, so developers who don't use it, have a lighter package.
https://guides.cocoapods.org/syntax/podspec.html#subspec

The HockeyApp SDK has a pretty good example of subspecs that remove dependencies, and set a default:
https://github.com/bitstadium/HockeySDK-iOS/blob/develop/HockeySDK.podspec